page-objects/pages/teamsPages/TMSAddEditTeamPage.js

```diff
@@ -9,7 +9,37 @@ const elementHelpers = require('../../../runtime/helpers/elementHelpers.js');
 
 const teamName = 'input[data-testid="team_name"]';
 const teamDescription = '#teamDesc';
-const orangeColor = "span[data-color='rgb(255, 173, 66)']";
+const teamColour = {
+	orangeColor: "span[data-color='rgb(255, 173, 66)']",
+	blueColor: "span[data-color='rgb(25, 118, 210)']",
+	greenColor: "span[data-color='rgb(46, 125, 50)']",
+	yellowColor: "span[data-color='rgb(255, 217, 90)']",
+
+};
+
+function getColorFromTable(color){
+	let colorItem = '';
+	switch(color){
+		case 'orange':
+		colorItem = teamColour.orangeColor;
+		break;
+		case 'blue':
+		colorItem = teamColour.blueColor;
+		break;
+		case 'green':
+		colorItem = teamColour.greenColor;
+		break;
+		case 'yellow':
+		colorItem = teamColour.yellowColor;
+		break;
+		default:
+			console.error(`This color: ${color} does not exist on the list of possible choices`);
+			break;
+		
+	}
+	return colorItem;
+}
+
 
 // submit button
 const createTeamBtn = 'button[data-testid="create_team_btn"]';
```

Comment:
Paste this please on top of the file where is the rest of the selectors.

Reply:
done